front/src/Data.elm

```diff
@@ -245,17 +262,12 @@ raspberrySyrup =
 
 raspberryLiqueur : Material
 raspberryLiqueur =
-    { name = "Raspberry liqueur", t = Sweetener }
-
+    { name = "Raspberry liqueur", t = Spirit }
 
-lemonPeel : Material
-lemonPeel =
-    { name = "Lemon peel", t = Garnish }
 
-
-limeSlice : Material
-limeSlice =
-    { name = "Lime slice", t = Garnish }
+orange : Material
+orange =
+    { name = "Orange", t = Garnish }
```

Comment:
Instead opted to do peels, slices, wedges, rinds, etc in quantities. And use the whole ingredients here.

Reply:
Makes sense! If you have an orange you have an orange peel :)